control/VersionedRequestFilter.php

```diff
@@ -32,7 +32,7 @@ public function preRequest(SS_HTTPRequest $request, Session $session, DataModel
 			$dummyController->popCurrent();
 			// Prevent output in testing
 			if(class_exists('SapphireTest', false) && SapphireTest::is_running_test()) {
-				return false;
```

Comment:
Happy to merge as-is, but my only comment here would be "can we make this the code execution path for regular requests as well as tests?"